package.json

```diff
@@ -3,26 +3,38 @@
 	"version": "0.8.0",
 	"author": "sheetjs",
 	"description": "Excel (XLSB/XLSX/XLSM/XLS/XML) and ODS spreadsheet parser and writer",
-	"keywords": [ "excel", "xls", "xlsx", "xlsb", "xlsm", "ods", "office", "spreadsheet" ],
+	"keywords": [
+		"excel",
+		"xls",
+		"xlsx",
+		"xlsb",
+		"xlsm",
+		"ods",
+		"office",
+		"spreadsheet"
+	],
 	"bin": {
 		"xlsx": "./bin/xlsx.njs"
 	},
 	"main": "./xlsx",
 	"dependencies": {
-		"ssf":"~0.8.1",
-		"codepage":"~1.4.0",
-		"cfb":">=0.10.0",
-		"jszip":"2.4.0",
-		"crc-32":"",
-		"adler-32":"",
-		"commander":""
+		"ssf": "~0.8.1",
+		"codepage": "~1.4.0",
+		"cfb": ">=0.10.0",
+		"jszip": "2.4.0",
+		"crc-32": "",
+		"adler-32": "",
+		"commander": ""
 	},
 	"devDependencies": {
-		"mocha":"",
-		"xlsjs":"",
-		"uglify-js":""
+		"mocha": "",
+		"xlsjs": "",
+		"uglify-js": "2.8.29"
```

Comment:
Explicitly using v2 here since v3.x uses different CLI commands which would break stuff in the makefile.  Decided this was the easier approach for now.